client/source/js/modules/model/calibration-ctrl.js

```diff
@@ -434,5 +439,32 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
       updateCharts($scope.parameters.cache.response);
     }, true);
 
+    $scope.reportDataEndError = function() {
+      return "End year must be more than "+ $scope.G.dataend + ".";
+    };
+
+    // Returns the value of the tallest chart.
```

Comment:
The idea is to use // for special explanations inside functions for example & jsdoc like comments for methods/modules and so on.